src/main/java/com/dutact/web/features/event/admin/dtos/EventUpdateDto.java

```diff
@@ -1,30 +1,31 @@
 package com.dutact.web.features.event.admin.dtos;
 
-import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.annotation.Nullable;
 import lombok.Data;
+import org.springframework.web.multipart.MultipartFile;
 
 import java.time.LocalDateTime;
 
 @Data
 public class EventUpdateDto {
-    @JsonProperty("name")
+    @Nullable
```

Comment:
can u add Optional for data type

Reply:
why?

Reply:
And do mapstruct and Spring OpenAPI support optional?

Reply:
Hmmm as search than they no support, nvm